middleware/TestingHandler.md

```diff
@@ -1,10 +1,16 @@
 # Testing Handler
-A piece of middleware that helps to mock different kinds of responses.
+A piece of middleware that helps to mock different kinds of responses. The goal of this testing handler is to allow a developer to test their application code with random errors injected into their application so that they can see whether the application handles server generated failures. Additionally, a developer can set manual responses to test specific conditions of our client library and the Graph API. 
 
 ## Objectives
-Provide a component, so that application developers can test their application with different kinds of responses (different response codes).
+Provide a configurable testing handler component that, by default, generates random error responses, and can be configured to generate tester specified errors.
 
 ## Requirements
+* MUST support response pass-through for non-intercepted responses.
```

Comment:
@darrelmiller @MIchaelMainer 
- If in case the request is a success scenario(in manual mode he would be manually mentioning that he needs 2xx response, in random mode it depends on the randomness), we pass the request to Graph and send back the response that we get, without any alterations.
- If in case the request is a failure scenario, then we directly return the response and don't pass the request to graph.

Reply:
There is no scenario where a developer needs to return success responses.